### PR TITLE
Entity tracking

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: src/ol3cesium.js [hunk 1]
```diff
@@ -207,6 +224,125 @@ olcs.OLCesium = function(options) {
...
+   */
+  this.boundingSphereScratch_ = new Cesium.BoundingSphere();
+
+  var eventHelper = new Cesium.EventHelper();
+  eventHelper.add(this.scene_.postRender, olcs.OLCesium.prototype.updateTrackedEntity_, this);
+};
+
+
+Object.defineProperties(olcs.OLCesium.prototype, {
+  trackedFeature: {
```
Comment (fredj) [inline src/ol3cesium.js]: add quotes to make sure the property is not renamed by the compiler ?

Comment (gberaudo) [inline src/ol3cesium.js]: There are no quotes in `src/core/olimageryprovider.js` so I did it the same here.
But I agree not quoting looks strange. I will check what the compiler actually generates.

Comment (fredj) [inline src/ol3cesium.js]: maybe the compiler is aware of `Object.defineProperties` and don't rename the keys

Comment (fredj) [inline src/ol3cesium.js]: (and +1 for using `Object.defineProperties` !)

**Comment by gberaudo** [comment]
Updated the PR and the example to use stacked views. This feature is currently limited to stacked views.

**Comment by gberaudo** [comment]
@fredj, could you please check the example and merge if you think it is good?

**Pull request merged**